demos/preemption_fairness/README.md

```diff
@@ -0,0 +1,324 @@
+# Kueue GPU Preemption Demo
+
+This repository demonstrates how Kueue preempts low-priority jobs to make room for high-priority jobs, ensuring fair resource allocation across teams in a Kubernetes environment.
+
+## Demo Recording
+
+To replay the demo locally (requires asciinema):
+
+```bash
+  asciinema play gpu-preemption-demo.cast
+```
+
+[Demo recording](https://asciinema.org/a/A46ZADXa9EQkoLxKM7NJD1Gu6) (hosted)
+
+## Overview
+
+This demo showcases Kueue's GPU preemption capabilities in a multi-team scenario:
+
+Team A: Has access to GPUs with high priority
+Team B: Shares the cluster but with lower priority
+Both teams belong to the same cohort and share CPU/memory quotas
+Team A can preempt resources from Team B when needed
```

Comment:
```suggestion
Team A: Has access to GPUs with high priority<br>
Team B: Shares the cluster but with lower priority<br>
Both teams belong to the same cohort and share CPU/memory quotas<br>
Team A can preempt resources from Team B when needed<br>
```
Alternatively you can use a bulleted list if you want this format for your README